src/frontend/js/components/Spinner/stories.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { Spinner } from './index';
+
+export default {
+  title: 'Components/Spinner',
+  component: Spinner,
```

Comment:
I'll customize the control for `theme` prop to be able to switch between `undefined` and `light` values or I'll use a checkbox to enable/disable `light`

```suggestion
  component: Spinner,
  theme: {
      control: 'inline-radio',
      options: [undefined, 'light'],
    },
```